.scripts/version/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bump_version",
+  "version": "1.0.0",
+  "description": "Version stamps all SDKs.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Azure/azure-iot-sdks"
```

Comment:
>"https://github.com/Azure/azure-iot-sdks" [](start = 11, length = 41)

this URIs are wrong now

Reply:
It's not clear how this URL is wrong.  What should we put instead?

---
In reply to: [265818897](https://github.com/Azure/azure-iot-sdk-node/pull/510#discussion_r265818897) [](ancestors = 265818897)

Reply:
We can amend at a later date per discussion

---
In reply to: [266112503](https://github.com/Azure/azure-iot-sdk-node/pull/510#discussion_r266112503) [](ancestors = 266112503,265818897)